x-pack/plugins/ml/public/application/jobs/new_job/job_from_lens/utils.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { Embeddable } from '@kbn/lens-plugin/public';
+
+export function getJobsItemsFromEmbeddable(embeddable: Embeddable) {
+  const { query, filters, timeRange } = embeddable.getInput();
+
+  // @ts-expect-error savedVis is private in Embeddable
+  const vis = embeddable.savedVis as LensSavedObjectAttributes;
```

Comment:
`savedVis` is private and so I shouldn't be doing this.
Is there a correct way to get the visualisation config from the embeddable?
Could I add a readonly `getSavedVis` function to the lens embeddable?